bbc_sale/models/product.py

```diff
@@ -67,6 +71,23 @@ def search_is_component(self, operator, value):
         return [('id', 'not in' if negate else 'in', templates.ids)]
 
     @api.multi
+    def compute_is_synced_magento(self):
```

Comment:
perform the search outside of the loop for better performance:
```
synced = self.env['magento.product.template'].search(
    [('erp_template_id', 'in', template.ids)]).mapped('erp_template_id')
for template in self:
    template.is_synced_magento = template.id in synced
```

Reply:
Fixed.